homeassistant/components/netatmo/climate.py

```diff
@@ -254,32 +242,36 @@ async def async_added_to_hass(self) -> None:
             )
 
         registry = await async_get_registry(self.hass)
-        device = registry.async_get_device({(DOMAIN, self._id)}, set())
+        device = registry.async_get_device({(DOMAIN, self._room.entity_id)}, set())
```

Comment:
No longer needed to pass empty set as 2nd arg.
```suggestion
        device = registry.async_get_device({(DOMAIN, self._room.entity_id)})
```